src/iscc/iscc.py

```diff
@@ -16,38 +16,35 @@
 ###############################################################################
 
 
-def meta_id(title, extra="", version=1):
+def meta_id(title, extra=""):
```

Comment:
Great, I was wondering exactly this :smile:

https://github.com/iscc/iscc-rs/blob/a3e3922c901fb645c7ec42cc9d0680a8b09dc20f/src/mid.rs#L9-L10